mmcv/image/io.py

```diff
@@ -60,6 +69,52 @@ def _jpegflag(flag='color', channel_order='bgr'):
         raise ValueError('flag must be "color" or "grayscale"')
 
 
+def _pillow2array(img, flag='color', channel_order='bgr'):
+    """Convert a pillow image to numpy array
+
+    Args:
+        img (:obj:`PIL.Image.Image`): The image loaded using PIL
+        flag (str): Flags specifying the color type of a loaded image,
+            candidates are `color`, `grayscale` and `unchanged`.
+        channel_order (str): Order of channel, candidates are `bgr` and `rgb`.
+
+    Returns:
+        np.ndarray: The converted numpy array
+    """
+    channel_order = channel_order.lower()
+    if channel_order not in ['rgb', 'bgr']:
+        raise ValueError('channel order must be either "rgb" or "bgr"')
+
+    if flag == 'unchanged':
+        array = np.array(img)
+        if array.ndim >= 3 and array.shape[2] >= 3:  # color image
+            array[:, :, :3] = array[:, :, (2, 1, 0)]  # RGB to BGR
+    else:
+        if img.mode == 'LA':
+            # When the mode is 'LA', the default conversion will fill in the
+            #  canvas with black, which sometimes shadows black objects in
+            #  the foreground.
+            #
+            # Therefore, a random color (124, 117, 104) is used for canvas
+            img = img.convert('RGBA')
+            canvas = Image.new('RGB', img.size, (124, 117, 104))
```

Comment:
(124, 117, 104) seems to be mean of images in RGB order. Is it OK if the image if in BGR order?